README.md

```diff
@@ -24,7 +24,7 @@ In addition to the notebooks listed in the following section, we have example ma
     * [Content-based Recommendations with FAISS, TFIDF, and Python](https://colab.research.google.com/github/arangodb/interactive_tutorials/blob/master/notebooks/arangoflix/similarMovie_TFIDF_ML_Inference.ipynb)
     * [Graph Neural Networks with PyTorch](https://colab.research.google.com/github/arangodb/interactive_tutorials/blob/master/notebooks/arangoflix/predict_Movie_Rating_GNN.ipynb)
     * [Matrix Factorization](https://colab.research.google.com/github/arangodb/interactive_tutorials/blob/master/notebooks/arangoflix/similarMovie_MF_ML_Inference.ipynb)
-* LangChain Integration: [Question-Answering with LLMs](https://colab.research.google.com/github/arangodb/interactive_tutorials/blob/master/notebooks/Langchain.ipynb)
+* LangChain Integration: [Question-Answering with LLMs](https://colab.research.google.com/github/arangodb/interactive_tutorials/blob/master/notebooks/example_output/Langchain_output.ipynb)
```

Comment:
I've updated the LangChain README link to point to the output file, as I believe it may useful for people to see what the `ArangoGraphQAChain` is capable of, at a quick glance.

I have no problem reverting this change however, just thought it would be convenient to have